src/basilisp/lang/runtime.py

```diff
@@ -1073,7 +1073,8 @@ def cons(o, seq) -> ISeq:
 
 def concat(*seqs: Any) -> ISeq:
     """Concatenate the sequences given by seqs into a single ISeq."""
-    return lseq.sequence(itertools.chain.from_iterable(filter(None, seqs)))
+    s = [to_seq(seq) if isinstance(seq, IPersistentMap) else seq for seq in seqs if seq]
+    return lseq.sequence(itertools.chain.from_iterable(s))
```

Comment:
```suggestion
    return lseq.sequence(itertools.chain.from_iterable(filter(None, map(to_seq, seqs))))
```

I believe this should be correct in all cases.

Reply:
Updated